model_demos/nlp_demos/falcon/utils/configuration_RW.py

```diff
@@ -1,3 +1,5 @@
+# Apache-2.0
```

Comment:
I don't think that we need to include any additional headers to this file as the original authors have already included the appropriate header.

Reply:
remove this license header